inc/ocf_mngt.h

```diff
@@ -237,6 +243,11 @@ struct ocf_mngt_cache_config {
 	 */
 	char name[OCF_CACHE_NAME_SIZE];
 
+	/**
+	 * @brief Cache name length
```

Comment:
Maybe let's use the comment from strncpy_s:
@brief The maximum number of characters to copy from name

Reply:
same comment applies to core config